droid-hal-device-img-boot.inc

```diff
@@ -138,8 +138,13 @@ fi
 %define factory_part_label fimage
 %endif
 
+%if 0%{!?external_media_devices:1}
+%define external_media_devices /dev/mmcblk[1-9]*p[1-9]*
```

Comment:
This is a bit risky thing to have here, as some devices have multiple eMMC chips which are mmcblk0 and mmcblk1. I wonder if we should have this by default empty or if we should do a bit more advanced detection e.g.

`[root@Sailfish ~]# cat /sys/class/block/mmcblk1/device/type   
SD
[root@Sailfish ~]# cat /sys/class/block/mmcblk0/device/type 
MMC
`

Reply:
Leaving it empty is probably only slightly less dangerous since the devices set in a new device configuration may well be copied from an existing device to start.

I added a check for the device type to the format script which will skip anything that's not an SD card in https://github.com/sailfishos/initrd-helpers/pull/12/commits/94be9667d93bd86af2a5f545c5236972e735e8b9. That should prevent accidents by people running it directly with bad arguments too.